sdk/core/azure-core/src/main/java/com/azure/core/implementation/annotation/FormParam.java

```diff
@@ -16,10 +16,10 @@
  *
  * <pre>
  * {@literal @}POST("spellcheck")
- * {@literal Observable<Response<ResponseBody>>} spellChecker(@Header("X-BingApis-SDK") String xBingApisSDK, @Query("UserId") String userId, @FormParam("Text") String text);</pre>
+ * {@literal Observable<Response<ResponseBody>>} spellChecker(@Header("X-BingApis-SDK") String xBingApisSDK, @Query
```

Comment:
Now that I see that this is prefixed with `<pre>`, we should fix this to look like how code. (ie. put the `@query` on the next line.`